File: VB/CodeBehind/XPO/PagedAsyncSource/Issues/Customer.vb
Imports DevExpress.Xpo

Namespace Issues
    Public Class User
        Inherits XPObject

        Public Sub New(ByVal session As Session)
            MyBase.New(session)
        End Sub

        Private _FirstName As String

        Public Property FirstName As String
            Get
                Return _FirstName
            End Get
            Set(ByVal value As String)
                SetPropertyValue(NameOf(User.FirstName), _FirstName, value)
            End Set
        End Property

        Private _LastName As String

        Public Property LastName As String
            Get
                Return _LastName
            End Get
            Set(ByVal value As String)
                SetPropertyValue(NameOf(User.LastName), _LastName, value)
            End Set
        End Property

        <Association("UserIssues")>
        Public ReadOnly Property Issues As XPCollection(Of Issue)
            Get
                Return GetCollection(Of Issue)(NameOf(User.Issues))
            End Get
        End Property
    End Class
End Namespace

File: CS/ViewModel/XPO/InstantFeedbackMode/Issues/Customer.cs
using DevExpress.Xpo;

namespace XPOIssues.Issues {
    public class User : XPObject {
        public User(Session session) : base(session) { }

        string _FirstName;
        public string FirstName {
            get { return _FirstName; }
            set { SetPropertyValue(nameof(FirstName), ref _FirstName, value); }
        }

        string _LastName;
        public string LastName {
            get { return _LastName; }
            set { SetPropertyValue(nameof(LastName), ref _LastName, value); }
        }

        [Association("UserIssues")]
        public XPCollection<Issue> Issues{
            get { return GetCollection<Issue>(nameof(Issues)); }
        }
    }
}
